src/crate/client/sqlalchemy/compiler.py

```diff
@@ -292,3 +292,8 @@ def limit_clause(self, select, **kw):
         Generate OFFSET / LIMIT clause, PostgreSQL-compatible.
         """
         return PGCompiler.limit_clause(self, select, **kw)
+
+    def for_update_clause(self, select, **kw):
+        # CrateDB does not support currently 'FOR UPDATE' statements.
+        # See https://github.com/crate/crate-python/issues/577
+        return ''
```

Comment:
Good point about the user warning, @matriv. I've just added a corresponding improvement with 154d77a298051.

I think it will be good like this. If you insist on adding it also to the documentation, I can take this, so we do not need to bother @surister about finding a good spot ;].

Reply:
I've amended the patch by adding a similar warning like on the other one.